package.json

```diff
@@ -25,7 +25,10 @@
     "mongoose": "^5.9.20",
     "passport": "^0.4.1",
     "passport-jwt": "^4.0.0",
+    "react-redux": "^7.2.0",
```

Comment:
I think u installed all redux libraries in serverside. because i can see mongoose in this file